nautobot_floor_plan/templates/nautobot_floor_plan/location_child_floor_plan.html

```diff
@@ -0,0 +1,16 @@
+{% extends "dcim/location.html" %}
```

Comment:
@glennmatthews When we start removing the object templates in favor of the Python template generation, will this be a problem? I am not asking DJ to change this, just curious how future changes in Nautobot would impact this.

Reply:
At some point, yeah, this will need to change to just being based on `generic/object_retrieve.html` (or adopt the UI Component Framework itself). For now we're keeping and just hollowing out the core templates (rather than deleting them entirely) to avoid exactly this problem in 2.x.